app/konnect/analytics/services-and-routes.md

```diff
@@ -16,11 +15,11 @@ You can generate and export a CSV file for:
 
 * A service, including daily requests by status codes for all versions of the
 service.
-* A service version, including a report of daily requests and status codes.
+* an API product version, including a report of daily requests and status codes.
 
 For a route, you can [view status codes](#view-performance-for-a-route)
 for a specified time frame but you can't export a route traffic report through
-Service Hub.
+API products.
```

Comment:
It's technically true that you can't export a route traffic report, but that's only because it's broken.  :grimacing:  This should get fixed soon as part of our work to harmonize our chart implementation.